Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM python:3.12-rc-alpine
+FROM python:3.12.5-alpine3.19
```

Comment:
Looked into this a bit more and the Chromium version on `3.12-alpine` is v127 while it is v124 on `3.12.5-alpine3.19`. This is a good decision to lock this version in just in case something like `3.12-alpine3.19` upgrades its Chromium version. It may need to be looked into more later, but for more this is a good decision.